examples/rl/cim/__init__.py

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT license.
 
 from .callbacks import post_collect, post_evaluate
-from .env_sampler import get_env_sampler
+from .env_sampler import agent2policy, get_env_sampler

```

Comment:
is the `agent2policy` a manually predefined mapping?

Reply:
Yes, I suppose so. Maybe @ysqyang  can confirm it.

Reply:
Yes